impeller/renderer/render_target.h

```diff
@@ -18,13 +19,16 @@ class Context;
 
 class RenderTarget {
  public:
-  static RenderTarget CreateOffscreen(const Context& context,
-                                      ISize size,
-                                      std::string label = "Offscreen");
-
-  static RenderTarget CreateMSAA(const Context& context,
-                                 std::shared_ptr<Texture> resolve_texture,
-                                 std::string label = "Offscreen");
+  static RenderTarget CreateOffscreen(
```

Comment:
In a later patch perhaps, we should consider creating a render target builder that makes it easy to set up render targets with commonly used configurations. The argument list here is getting out of hand already.

Reply:
Agreed, added a bug for this: https://github.com/flutter/flutter/issues/104006